frontend/javascripts/components/large_table_wrapper.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import { Table } from "antd";
+import * as React from "react";
+
+type Props = {
+  children: React.Node,
+  expandedRowRender?: Function,
```

Comment:
I think we should make this non-optional, since the table should only be used for the fixed column + expandable rows case. Then, `hasExpandableRows` can be removed since it will always be true.

Consequentially, this component could be renamed to `FixedExpandableTable`. This makes it more clear, when it should be used.